apps/bpytop.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# T&M Hansson IT AB © - 2020, https://www.hanssonit.se/
+
+# shellcheck disable=2034,2059
+true
+SCRIPT_NAME="BPYTOP"
+SCRIPT_EXPLAINER="BPYTOP is an amazing alternative to ressource-monitor software like top or htop."
+# shellcheck source=lib.sh
+source /var/scripts/fetch_lib.sh || source <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
+
+# Check for errors + debug code and abort if something isn't right
+# 1 = ON
+# 0 = OFF
+DEBUG=0
+debug_mode
+
+# Must be root
+root_check
+
+# Needed for snaps to run
+install_if_not snapd
+
+# Check if bpytop is already installed
+if ! snap list | grep -q bpytop
+then
+    # Ask for installing
+    install_popup "$SCRIPT_NAME"
+else
+    # Ask for removal or reinstallation
+    reinstall_remove_menu "$SCRIPT_NAME"
+    # Removal
+    sed -i "s|.*bpytop'||g" /home/"$UNIXUSER"/.bash_aliases
+    sed -i "s|.*bpytop'||g" /root/.bash_aliases
+    snap remove bpytop
+    # Show successful uninstall if applicable
+    removal_popup "$SCRIPT_NAME"
+fi
+
+# Install it!
+if yesno_box_yes "Do you want to install BPYTOP?" "$SUBTITLE"
```

Comment:
maybe this is not needed?

Reply:
Yes, this is not needed. It is already handled by the install_popup